nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -368,10 +350,7 @@ def forward(self, x, pad_mask=None, cache=None, cache_next=None):
         if pad_mask is not None:
             x = x.float().masked_fill(pad_mask.unsqueeze(1), 0.0)
 
-        if cache is not None:
-            x = self.depthwise_conv(x, cache=cache, cache_next=cache_next)
-        else:
-            x = self.depthwise_conv(x)
+        x, cache = self.depthwise_conv(x, cache=cache)
```

Comment:
We need to check if cache is None or not and based on that decide on the number of outputs.